packages/core/content-releases/admin/src/components/ReleaseActionOptions.tsx

```diff
@@ -0,0 +1,89 @@
+import * as React from 'react';
+
+import {
+  FieldInput,
+  FieldLabel,
+  VisuallyHidden,
+  Field,
+  type FieldProps,
+} from '@strapi/design-system';
+import styled from 'styled-components';
+
+interface FieldWrapperProps extends FieldProps {
+  actionType: 'publish' | 'unpublish';
+}
+
+const FieldWrapper = styled(Field)<FieldWrapperProps>`
+  border-top-left-radius: ${({ actionType, theme }) =>
+    actionType === 'publish' ? theme.spaces[1] : theme.spaces[0]};
+  border-top-right-radius: ${({ actionType, theme }) =>
+    actionType === 'publish' ? theme.spaces[0] : theme.spaces[1]};
+  border-bottom-left-radius: ${({ actionType, theme }) =>
+    actionType === 'publish' ? theme.spaces[1] : theme.spaces[0]};
+  border-bottom-right-radius: ${({ actionType, theme }) =>
+    actionType === 'publish' ? theme.spaces[0] : theme.spaces[1]};
+
+  > label {
+    color: inherit;
+    padding-top: ${({ theme }) => theme.spaces[2]};
+    padding-bottom: ${({ theme }) => theme.spaces[2]};
+    padding-left: ${({ theme }) => theme.spaces[3]};
+    padding-right: ${({ theme }) => theme.spaces[3]};
+    text-align: center;
```

Comment:
same here, just to reduce the number of style lines
```
padding: ${({ theme }) => theme.spaces[2] theme.spaces[3]};
```

Reply:
sure could do that